Automattic/sensei
# Add woocommerce auto-installation for some features from Setup Wizard

Part of #3139

### Changes proposed in this Pull Request

* Introduces auto-installation of WooCommerce when Sensei WC Paid Courses or Sensei Content Drip is selected.

### Testing instructions

* Make sure the `setup_wizard_all_extensions` feature flag is enabled.
  * If not, you can add the following code to your `wp-config`:
    ```php
    define( 'SENSEI_FEATURE_FLAG_SETUP_WIZARD_ALL_EXTENSIONS', true );
    ```
* Ensure you don't have WooCommerce installed.
* Install Sensei WC Paid Courses / Sensei Content Drip, but keep them deactivated. (It's because the installation through WooCommerce will be part of another PR)
* Go to the Setup Wizard.
* Navigate to the Features step.
* Select the Sensei WC Paid Courses and install. WooCommerce should be automatically selected and installed together.
* Go to the Features step again and install the Sensei Content Drip. WooCommerce should not be selected again because it's already installed.
